guarddog/analyzer/sourcecode/npm-obfuscation.yml

```diff
@@ -0,0 +1,38 @@
+rules:
```

Comment:
This rule looks great, any clue about the expected FP rate? Sounds like it would be great to scan the top 1k packages manually, and then maybe a small random sample?

Reply:
Yes, I scan a random sample of 100 top packages manually.
But I can, download 1k of the top packages and then scan last week.
I will let you know the outcome

Reply:
very nice!

Reply:
Scanned the 1k packages. 0 FP 🙌 
While scanning this week I found actual malware, so 👍